components/hero/Hero.jsx

```diff
@@ -0,0 +1,20 @@
tzoJV4vEMGG7/Lnfn07SbQx6nNiVa1szkOo4cflPO8mc5ZNJ0vdaNOEqTQqopHDnrlobRO5JJ3d0WN+N9b31MjOx4Ttwy5RTBFt6C3ITAQ6DG+ccqoCbidGqC1Tog12FpyzxZtEVa5VfyU7qDzb+YjpkB5GdWq0hpaUtXw7hj/7CD+kFf8AxgltUhcsJTjxTOVWhlbze4/wjRI9wsh7ll9Dn6GZLaOynomzjLgwzU/Q9DJuLRwZPHnBW1r7RUukEedYIlEDIbsJhaLY8TGCSTlHWzoSsK5Jymg8N4x6TgI1mvcX0PNSOR/aU9JAIrezFja2d+Uzd6Q0ZuMk0dj2fi0rLcZMPzKdVP7jrFVsNMPsTayvZWbcqDIEG292PPpNPS2nUXJrOPRvXQ+kR7Oq1LaCxeHylPg8Kq1g4FjZhfvn+00gxSOP0nk2Xx0kCth7G4ir4uxGmnsGJTeWwl/4Mx25TFF8ipO6eYJJlFQfh5fU+ksEpjI6G06MuP8AkhSe/R2wlFx4s3yutr3jLuW6DlMtszFhHBctugHQkrc5Alb95qaNZHXeUgjpI4VkSfNUJJcf2MskYqoJJqtItRxOhCqQwMQ6G6nyOn9pY4LaaP7p91v0nj2PGU+IeVOKRtVJBBuCNbiHin0WUYy70bk5Q1OUg7LxftVDMCGWwYdeY5iSajyf6I1TphOwjLuJEx+NCIzWZt0E7qLvObcFXiek51W/EN6rslNPZgZAud5vNRkp6Zw6W2LLJGCtnQsRVUXv8ZmNrbWpEMn57i1h+W/UzI4raNRz77s3QnL/ACjKCk98gIOf0c8vNdVFf5BUSCT6eAduFoInKP2cGzmjEtHaaWgRbRRMLfpF2/QGaEoiVzj6LA9GAiy1wu26qe7ffUcHz9DrIAHCEyxLGTaNDT8XKMmpn/tYH4ECKqeLUt7tNr9SF+V5kmTOHHpMPOX2azZ233dySgC8bEk+s1OBxyuMj3HH0mD2HbdPf+0twg84jyuDo6oqXFM2iVL3kzZ7OHBRrcSL6jtxmf8ADNW6NvktZrC+dhYTQtQVheJPymukUTbRa1MQSYjMyNg8WFIRza+SsdD0JOh+ctyltAI8M3L0KmRUoX0H33jq4Nf5iB0AvJSVDFHOXUmG5BJWVBuqptGqlQnjbtHNwdIRS2s1A37Ij0gZj/FnhRK96iWWsNGAtvW0Dgfm5b2o6gWm8WmDpYyPiKHSMkMqlqSPPWJq1Uf2dS4ZTYg8PPlxyyIM2eCxaIi7o3mtLDxv4bFRS6i1RdOo13T+3KYrAbfNFSpS7DLPhaQzQl/ScmfA4PS0a1cRXqaDdHWCZSr4tqNwEE5v4J/RDiyg9oIN68atHkE7Wkh2kh1RHVMbRY8slJgFoSOveK3+nxgAyhWiGGanORqj3jtZ7mw0jYS8rHXZibsqvuZE5H4S3/iwATeZ8jhDBiyipOy0fIcVTVm78I4wMHHJwfIgfQzY4apu/wBJ+B59py7wzitysB+sW8xmP3HnOh4avz0m4KSplcUuStlnjMMHUqdDM4PEFfAOEqhquGb8h/nTmqsdRyVvIgCaLDVLjdJ6r25eUjY/CpUQo4DK2RH3pEjFwdGnBvrs0Gy9qUcQm/RcOvG2RU8mXVT3ksvOHY7C1sFW3qbut/yOpsSP0twNuRyM0GzPxBrLYVkWoP1L7jemak+kvGS9k45uLqXZ081RC9tyPkdJlsF4zw1SwLlDycW/1C6/GXNPFK4ujqw5qwI9RKJpl4zjLpkx3U65H4Su2jjjQUszhVGpb8ovkLknK8eZzI1ezKysA6MCGVhcEHUWMZL6LJX+Jldq/iBhrFdz2h090ED1b9rzmW08WKtQuF3QeF7+ptnLXxZsH+HqkoCaTZqTc7t/5CenA8R5zOxZNvs5c2TI/jIKCCCAgLQSQgjCR5AYkhWPJHFEaRukcWoOok2mKOiRsRX4DzMTXr3yGnzgSnbM+Q+sMY1thEpT4n04x4LxhqVGZzP3xiHqXytlM7bM2IvFiJ8oodoRBynUKkMNQQR3E6BsvHh0BHETng7S32Fj9xgrflJ9LzLRXDLjKmb9MQRYjUZiXG+GUOuhF5mFqka58jzEtNk4rModDmvQ8R5x3G0ei1yiFtfZy16bI3dTxVuBnMsRRam7I4symx/YjpxnX2SYfx9g90pWA19xj/qX/d8IjWjkz47jy9ozNNxLXB0najVNIsrKAwKEq1xc5EZ8JSIQZf7F2h7JLW1zv8JFunZxxVMqsL4yxiZe1Ljk4DfHX4y1o/iFVtZ6SN1VmT53mY2vh92oSB7re8PPX4yvnXF2rR1RySXTNdtTxaKyMvsyLi2bgj5ZzJmJghbsM8kp1yfQII9SWCLZOxtDJSmRBHlNoJIWRIUxl3vC3ico9TS1+cToAES1idYtz/bz+zCYxA084O9gYFhssJYoCEUCxQGUSRDBgMKEUmRiVhrAA1GxtqggUqht+hz/AOJ+ss6jlT8iJihnHUxbqLBzbgDnaOpHVh8njqR1PZm0BUWxI3xr16yp8cIDhn6FSPJh9TMPQ2w6MGBzHKSds+JqmIT2bKqjK5BNzY3+kZtNFp5scouikorcgS7ROEqdnr73kZd07Tlyd0cJC2vhSae9+nPyORmbm/qqrYete2SMR6ZTAS+H8aHiCOU6d4gCTEUKLnWUboLY4ihRcwpEq1SxhROAKGwIsREUsdhY9Ta0fBkdTFI0SSFHjCAy84neikOsCAxLCKVoLwW5QihiGIkQwYDCul4pYkwxrAAeWEYLwATUYZcxDR1hGQhJsJgol4Aanyk01DzjVGnuqBHCJzydysYaxBLKRfUShIztNDuytxlMK29zzlsMq0GLGUUKLnWMVKhJuYHe8RLpe2MkCCCCEIIBBBMYdSKEEEViMcHCKp6wQRACn1MSIIIRQzCWCCAI5DOsEEAo40Cw4ITCKkewoyhwRZdDokLHBBBIGFCVO2vzL2PzgglMX5Bj2VkEEE6hwQQQTGP/2Q=="
```

Comment:
For image, please create a folder in the public folder called images and store it there. Import the image then to make the component more readable.